### PR TITLE
Wdfn 48 see available data

### PR DESCRIPTION
**Pull request by ayan-usgs**

### CODE REVIEW

File: waterdata/views.py [hunk 1]
```diff
@@ -49,8 +49,11 @@ def monitoring_location(site_no):
                                                           'siteStatus': 'all',
                                                           'agencyCd': agency_cd})
         if parameter_data_resp.status_code == 200:
-            param_data = parse_rdb(parameter_data_resp.iter_lines(decode_unicode=True))
-            location_capabilities = get_capabilities(param_data)
+            param_data = [param_datum for param_datum in
+                          parse_rdb(parameter_data_resp.iter_lines(decode_unicode=True)) if param_datum['parm_cd']]
+            site_dataseries = [get_disambiguated_values(param_datum, app.config['NWIS_CODE_LOOKUP'], {}) for
+                               param_datum in param_data]
+            location_capabilities = set([param_datum['parm_cd'] for param_datum in param_data])
```
Comment (danielnaab) [inline waterdata/views.py]: set can take a generator, so you can avoid the intermediate list: `set(param_datum['parm_cd'] for param_datum in param_data)`

File: waterdata/templates/monitoring_location.html [hunk 2]
```diff
@@ -20,6 +20,47 @@ <h1>Station Name: {{ station.station_nm }}</h1>
...
+                                        <th scope="col">SRS ID</th>
+                                        <th scope="col">Start Date</th>
+                                        <th scope="col">End Date</th>
+                                        <th scope="col">Record Count</th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    {% for series in site_dataseries %}
+                                        <tr>
+                                            <td>{{ series.parm_cd }}</td>
```
[the remark this answers is not in the document]
Comment (ayan-usgs) [reply waterdata/templates/monitoring_location.html]: Sounds good.

File: waterdata/views.py (hunk 1 above)
Comment (ayan-usgs) [reply waterdata/views.py]: Okay.

File: waterdata/tests/commands/lookup_generation/test_commands.py [hunk 3]
```diff
@@ -21,14 +21,14 @@ def test_help():
...
-    assert result.exit_code == 0
-    assert os.path.exists(os.path.join(app.config['DATA_DIR'], 'nwis_lookup.json'))
+# def test_nwis(app, fs, nwis_mock):
+#     fs.CreateDirectory(DATA_DIR)
+#
+#     runner = CliRunner()
+#     result = runner.invoke(manage.cli, ['generate_lookups', '--nwis'])
+#
+#     assert result.exit_code == 0
+#     assert os.path.exists(os.path.join(app.config['DATA_DIR'], 'nwis_lookup.json'))
```
[the remark this answers is not in the document]
Comment (ayan-usgs) [reply waterdata/tests/commands/lookup_generation/test_commands.py]: No... I did not intend to check that in. :flushed: 

Thanks for catching that.

File: waterdata/commands/lookup_generation/__init__.py [hunk 4]
```diff
@@ -58,6 +98,7 @@ def generate_lookup_file(datadir, filename='nwis_lookup.json'):
         if 'params' in lookup_config:
             params.update(lookup_config.get('params'))
         resp = execute_get_request(CODE_HOST_ENDPOINT, path=lookup_config.get('urlpath'), params=params)
+        print(resp.url)
```
Comment (mbucknell) [inline waterdata/commands/lookup_generation/__init__.py]: Should probably remove the print statement

Comment (ayan-usgs) [reply waterdata/commands/lookup_generation/__init__.py]: Yep.

**Pull request merged**